app/build.gradle.kts

```diff
@@ -187,6 +187,9 @@ dependencies {
     kaptAndroidTest(Libraries.Hilt.compiler)
     androidTestUtil(TestLibraries.testOrchestrator)
 
+    debugImplementation("androidx.compose.ui:ui-tooling:1.1.1")
+    implementation("androidx.compose.ui:ui-tooling-preview:1.1.1")
```

Comment:
Shall we bump them to a Version variable?

Reply:
ops didnt see it sneak in,sure will do